### PR TITLE
Added Graphics Meetup 3 post

### PR DESCRIPTION
**Pull request by AngelOnFira**
A post for the upcoming Rust Graphics Meetup

cc @kvark @cwfitzgerald

### CODE REVIEW

File: content/blog/graphics-meetup-03/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+**16:40 - 16:55**
+
+> Quick introduction to Rafx, and some ideas for making more extensible renderers
+
+### Rend3: High Performance, Cross Platform, GPU Driven Rendering in wgpu and WebGPU | [Connor Fitzgerald]
+
+**16:55 - 17:25**
+
+> An overview of the current state of rend3: new data and gpu driven rendering
+> model, performance.
```
Comment (cwfitzgerald) [inline content/blog/graphics-meetup-03/index.md:49]:
```suggestion
> An overview of the current state of rend3: new data and gpu driven rendering model, 
> performance improvements, and a look into the future.
```
Missed a line

Comment (AngelOnFira) [reply content/blog/graphics-meetup-03/index.md]: Thanks!

**Review by cwfitzgerald — APPROVED** [review]
LGTM

**Comment by AngelOnFira** [comment]
Sorry, overriding the block so I can get this out in time to advertise 😅

File: content/blog/graphics-meetup-03/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+**16:05 - 16:40**
+
+> Introducing a lean and mean graphics library 'Blade', for fun and no profit.
+
+### Implementing an Extensible Renderer | [Philip Degarmo]
+
+**16:40 - 16:55**
+
+> Quick introduction to Rafx, and some ideas for making more extensible renderers
```
Comment (AngelOnFira) [inline content/blog/graphics-meetup-03/index.md:42]:
```suggestion
> Quick introduction to Rafx, and some ideas for making more extensible renderers.
```

**Pull request merged**